Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-dbt-databricks = "1.2.1"
+dbt-databricks = "1.3.0"
```

Comment:
Let's make this update in a separate PR. This change needs to be coordinated with upgrading the GH action runner.

Reply:
Sounds good 👍